Examples/UtiliKit-iOSExample/Container/ContainerExampleViewController.swift

```diff
@@ -21,26 +23,47 @@ class BaseContainerViewController: UIViewController {
     private lazy var controllerA = ViewControllerA()
     private lazy var controllerB = ViewControllerB()
     
+    private var interactionController: HorizontalPanGestureInteractiveTransition?
+    private var animationController = WipeTransitionAnimator(direction: .leftToRight)
```

Comment:
What is a WipeTransition? Did you mean Swipe?

Reply:
A WipeTransition in this case, is in an interactive transition where you can pan from the screen edge to reveal the destination view controller.